src/utils/token-operations.util.ts

```diff
@@ -1,57 +1,46 @@
-import { from } from 'rxjs';
-import { map } from 'rxjs/operators';
-
 import { tzktApi } from '../api.service';
+import { OPERATION_LIMIT } from '../config/general';
```

Comment:
You are using only `data` property of the returned value of functions below.   
So, you could map the result in this file and return prepared `data`